server/models/Thread.model.js

```diff
@@ -0,0 +1,20 @@
+const { Schema, model } = require("mongoose");
+
+const threadSchema = new Schema(
```

Comment:
I realized we'll want to add another field and association here. We'll need a `thread` to belong to a `lesson` (or `TeachingText`/`VideoText`). So when I fetch a lesson, I can also fetch the threads associated with it.

Reply:
You're right, I commit that, for the moment I linked to the text model